### PR TITLE
Implemented PDF exporting at grid creation

### PR DESCRIPTION
**Pull request by gabormagyar**
Sample PDF:
[Lab_Kod_Meno.pdf](https://github.com/vacuumlabs/covidhack/files/4515590/Lab_Kod_Meno.pdf)

### CODE REVIEW

File: utils/pdf/pdf.ts [hunk 1]
```diff
@@ -575,13 +576,29 @@ export const getJournalContent = (entries: Array<OfficeJournalProps> = [{}, {},
...
+      {
+        style: 'table',
+        table: {
+          // eslint-disable-next-line @typescript-eslint/no-unused-vars
+          widths: grid[0].map((_) => '*'),
+          heights: grid.map((_) => 25),
+          body: grid.map((r) => r.map((c) => c.value)),
+        },
+      },
+    ],
```
Comment (paldys) [inline utils/pdf/pdf.ts:592]: If you add this here, you don't have to change `createPdf`'s signature. (and it makes more sense to enforce this at creation)
```suggestion
    ],
    pageOrientation: 'landscape',
```

Comment (gabormagyar) [reply utils/pdf/pdf.ts:593]: Good catch! Fixed in the next commit.

File: pages/create-lab-result.tsx [hunk 2]
```diff
@@ -176,18 +178,20 @@ const CreateLabResult = () => {
...
-                  color="primary"
-                  disabled={submitting}
-                  startIcon={submitting && <LoadingIcon style={{color: 'white'}} size={20} />}
-                >
-                  Začať test
-                </Button>
-              </div>
+
+            <div className="button-panel">
+              <IconButton onClick={() => createPdf(title, getGridContent(title, grid))}>
```
Comment (Siegrift) [inline pages/create-lab-result.tsx:183]: use `title || 'grid'` as with empty title it's called 'file'

File: pages/create-lab-result.tsx [hunk 3]
```diff
@@ -176,18 +178,20 @@ const CreateLabResult = () => {
...
+                <PictureAsPdfIcon />
```
Comment (Siegrift) [inline pages/create-lab-result.tsx:184]: Let's add text: `Mriežka` as well.

File: pages/create-lab-result.tsx (hunk 2 above)
Comment (gabormagyar) [reply pages/create-lab-result.tsx]: This is done

**Comment by gabormagyar** [comment]
> There is now the cell status feature, which is not in the pdf. We could write the name (or abbreviation) of the cell in the pdf cell. Can you ask Palo about this?

We should discuss how to best do this. Colors are probably not a good idea because of printing, abbreviations should probably work.

File: pages/create-lab-result.tsx (hunk 3 above)
Comment (gabormagyar) [reply pages/create-lab-result.tsx]: Done

**Pull request merged**